modules/installation-user-infra-generate-k8s-manifest-ignition.adoc

```diff
@@ -96,6 +103,19 @@ Because you create and manage the worker machines yourself, you do not need
 to initialize these machines.
 endif::aws,gcp[]
 
+ifdef::osp[]
+. Remove the Kubernetes manifest files that define the control plane machines and compute machineSets:
++
+----
+$ rm -f openshift/99_openshift-cluster-api_master-machines-*.yaml openshift/99_openshift-cluster-api_worker-machineset-*.yaml
+----
++
+Because you create and manage these resources yourself, you do not need
+to initialize them.
++
+* You can preserve the machineSets files if you want to create compute machines via the machine API, but you need to update references to them to match your environment.
+endif::osp[]
+
 . Modify the `manifests/cluster-scheduler-02-config.yml` Kubernetes manifest file to prevent Pods from being scheduled on the control plane machines:
```

Comment:
Perhaps the Upstream Python script can be included here
https://github.com/openshift/installer/blob/master/docs/user/openstack/install_upi.md#make-control-plane-nodes-unschedulable

Reply:
I considered it! Directly modifying the file seems pretty straightforward, though. Open to feedback on this, though.